5_data_munge/src/munge_wqp_files.R

```diff
@@ -73,7 +73,7 @@ munge_wqp_withdepths <- function(in_ind, min_value, max_value, max_daily_range,
 
 munge_wqp_withoutdepths <- function(in_ind, min_value, max_value, max_daily_range, out_ind) {
   dat <- readRDS(sc_retrieve(in_ind, remake_file = 'getters.yml')) %>%
-    select(MonitoringLocationIdentifier, ActivityStartDate, ResultMeasureValue,
+    select(MonitoringLocationIdentifier, ActivityStartDate, ResultMeasureValue, StatisticalBaseCode,
```

Comment:
Just confirming for myself that you specifically retain the `StatisticalBaseCode` column for the subset of data without depths, and that this column is implicitly retained for the subset of data with depths (because there's no analogous `select` step in `munge_wqp_withdepths()`). Is that what you expect as well?

Reply:
I ignored "withdepths" because those data are actually dead-ended in this pipeline. I don't totally remember the history, but I recall there was some concern that some of that data wasn't from streams, and was likely from lakes/reservoirs. It represents a pretty smallish fraction of the total WQP data at this stage of processing (<8%), so I'm okay continuing to ignore these data for now. But I made [an issue](https://github.com/USGS-R/2wp-temp-observations/issues/61) to return to these as some point!

Reply:
I agree with the reservations you have about the data being from streams, as well as your decision to filter out these records with depths. Thanks for creating that issue as a reminder!